graphql/user/fields.ts

```diff
@@ -86,7 +86,7 @@ export class UserFieldsResolver {
     @FieldResolver((returns) => [String])
     @Authorized(Role.ADMIN)
     async roles(@Root() user: User) {
-        const fakeContext: GraphQLContext = { ip: "?", prisma, sessionToken: "fake" };
```

Comment:
Seems like we need a commit hook, this made it to master with wrong formatting (shame on me :sweat_smile: )